src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -24,26 +24,52 @@ export interface CrumbProps
       | "disabled"
     >,
     TagProps {
-  /** This sets the Crumb to current, does not render Link */
   isCurrent?: boolean;
+  onClick?: (
+    ev: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>,
+  ) => void;
 }
 
-const Crumb = React.forwardRef<HTMLAnchorElement, CrumbProps>(
-  ({ href, isCurrent, children, onClick, ...rest }: CrumbProps, ref) => {
+const Crumb = React.forwardRef<HTMLLIElement, CrumbProps>(
+  ({ href, isCurrent, children, onClick, ...rest }, ref) => {
     const { isDarkBackground } = useBreadcrumbsContext();
 
+    const isSafari = React.useMemo(() => {
```

Comment:
comment: We should avoid browser sniffing, having this work on all browsers isn't going to break anything etc